chapter_9/ex9.rb

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env ruby
+
+# Here's some new strange stuff, remember type it exactly.
+
+days = 'Mon Tue Wed Thu Fri Sat Sun'
+months = "Jan\nFed\nMar\nApr\nMay\nJun\nJul\nAug"
+
+puts "Here are the days: #{days}"
+puts "Here are the months: #{months}"
+
+puts <<-TEXT
```

Comment:
This would also work:
```
puts "
There's something going on here.
With the three double-qoutes.
We'll be able to type as much as we like.
Even 4 lines if we want, or 5 or 6.
"
```